.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 ### All Submissions:
 
-* [ ] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
-* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
+* [ ] Have you followed the [WooCommerce Contributing guideline](https://github.com/ClassicPress-research/classic-commerce/blob/master/.github/CONTRIBUTING.md)?
+* [ ] Does your code follow the repo standards?
```

Comment:
Do you plan to change the coding standards for this repo? (I wouldn't recommend it, in which case you might want to leave the link to the WP standards for now...)

Reply:
No plans to change these anytime soon. We will be leaving the links inset.